icommons_ext_tools/settings/local.py

```diff
@@ -60,6 +60,11 @@
      }
  }
 
+DATABASE_ROUTERS = ['icommons_common.routers.DatabaseAppsRouter']
+DATABASE_APPS_MAPPING = {
+    'canvas_course_site_wizard': 'default',
+}
+DATABASE_MIGRATION_WHITELIST = ['default']
```

Comment:
This may be more than you are looking to do right now, but can we move database config to base.py?